dask/base.py

```diff
@@ -1,9 +1,15 @@
 import warnings
 from operator import attrgetter
+from hashlib import md5
+from functools import partial
 
-from toolz import merge, groupby
+from toolz import merge, groupby, curry
+from toolz.functoolz import Compose
 
 from .context import _globals
+from .utils import Dispatch, ignoring
+
+__all__ = ("Base", "compute", "normalize", "tokenize")
```

Comment:
The term `normalize` is fairly general for a fairly specific and internal operation.  Perhaps `normalize_token`?